build-npm-module/index190.js

```diff
@@ -17,6 +17,8 @@ module.exports = {
   getProperCase
 };
 
-// Now you have finished your case-coverter module. Let's publish it to NPM.
-// Normally you would need to create your own account on npmjs.com.
-// But for now, just use this demo account: username: camper, password: camper
+/*
+Now that you have finished your case_converter module, let's publish it to the npm registry.
```

Comment:
This test description should come later when learners have to enter `npm adduser`. It might be better to have students edit the `README.md` file, then login before publishing the package. Either way, here's the edited text for this step:


Now that you have finished your `case_converter` module, let's publish it to the npm registry.
Normally you would need to create your own account on https://www.npmjs.com.
But for now, just use this demo account: 
```
username: camper
password: camper
```